.github/workflows/tests.yml

```diff
@@ -52,13 +52,13 @@ jobs:
         key: ${{ hashFiles('src/**', 'lualib/**', '.github/workflows/tests.yml', 'kong/.requirements') }}
 
     - name: Install packages
-      if: steps.cache-deps.outputs.cache-hit != 'true'
+      # if: steps.cache-deps.outputs.cache-hit != 'true'
```

Comment:
Should we remove these `#` comments?

Reply:
I found the build cache is not working well, so I commented on it since this repo is not built frequently, how about letting it build every time?

Reply:
This should work, if cache needs purging, you can do it at https://github.com/Kong/lua-kong-nginx-module/actions/caches. Can we put this back for now?

Reply:
put it back